### PR TITLE
prevent NoneType exception when profiling empty datasets

### PR DESCRIPTION
**Pull request by sgomezvillamor**
`res["unexpected_percent"]` is `NoneType` when profiling empty datasets and the division causes an exception. This PR is preventing that scenario.

Issue https://github.com/linkedin/datahub/issues/3132

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py [hunk 1]
```diff
@@ -219,7 +219,8 @@ def _handle_convert_column_evrs(  # noqa: C901 (complexity)
             elif exp == "expect_column_values_to_not_be_null":
                 column_profile.nullCount = res["unexpected_count"]
                 if "unexpected_percent" in res:
```
Comment (shirshanka) [inline metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py:221]: I think the system can handle `column_profile.nullProportion = None`, so how about only setting it only when both of these are true. 

So I guess: 
```
if "unexpected_percent" in res and res["unexpected_percent"] is not None:
      column_profile.nullProportion = res["unexpected_percent"]
```

**Comment by shirshanka** [comment]
@sgomezvillamor : seems like lint is complaining (https://github.com/linkedin/datahub/pull/3144/checks?check_run_id=3402002680) 

You can fix it via: `./gradlew :metadata-ingestion:lintFix` from the top level directory. 

Also I had a suggestion for rewriting the check and combining it with the previous.

File: metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py (hunk 1 above)
Comment (sgomezvillamor) [reply metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py:221]: That makes totally sense. I will try that!

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**